frontend/pages/policies/ManagePoliciesPage/ManagePoliciesPage.tsx

```diff
@@ -105,15 +118,70 @@ const ManagePolicyPage = ({
   const [showPreviewPayloadModal, setShowPreviewPayloadModal] = useState(false);
   const [showAddPolicyModal, setShowAddPolicyModal] = useState(false);
   const [showDeletePolicyModal, setShowDeletePolicyModal] = useState(false);
-  const [showInheritedPolicies, setShowInheritedPolicies] = useState(false);
 
   const [teamPolicies, setTeamPolicies] = useState<IPolicyStats[]>();
   const [inheritedPolicies, setInheritedPolicies] = useState<IPolicyStats[]>();
 
+  const initialSearchQuery = (() => {
+    let query = "";
+    if (queryParams && queryParams.query) {
+      query = queryParams.query;
+    }
+    return query;
```

Comment:
would `return queryParams?.query ?? ""` accomplish the same thing?

Reply:
Cleaned these up, thank you!